src/simulation/point.py

```diff
@@ -215,4 +236,13 @@ def pour_from_buffer(self):
         self.viscosity_dynamic = (sum([tup[0] * tup[1] for tup in self.oil_buffer]) + self.oil_mass * self.viscosity_dynamic) / oil_mass
         self.emulsification_rate = (sum([tup[0] * tup[2] for tup in self.oil_buffer]) + self.oil_mass * self.emulsification_rate) / oil_mass
         self.oil_buffer = []
-        self.oil_mass = oil_mass
\ No newline at end of file
+        self.oil_mass = oil_mass
+
+    def add_oil_to_other(self, coord: Coord_t, mass: float) -> None:
```

Comment:
Stylish proposition:
pour instead of add - because right now it's not only adding it's also removing oil from its self